references/classification/train_quantization.py

```diff
@@ -22,6 +29,17 @@ def main(args):
     if args.post_training_quantize and args.distributed:
         raise RuntimeError("Post training quantization example should not be performed on distributed mode")
 
+    # Validate quantization workflow type
+    quantization_workflow_type = args.quantization_workflow_type.upper()
+    if quantization_workflow_type not in QuantizationWorkflowType.__members__:
+        raise RuntimeError(
+            "Unknown workflow type '%s', please choose from: %s"
+            % (args.quantization_workflow_type, str(tuple([t.lower() for t in QuantizationWorkflowType.__members__])))
+        )
+    use_fx_graph_mode_quantization = (
```

Comment:
This will be simplified drastically if we change to a boolean flag as mentioned above. The Enum class can be removed and so does this additional code with:
```
use_fx_graph_mode_quantization = args.use_fx_quantization
```

Reply:
note: the product name is "FX graph mode quantization".  There may be other quantization workflows that use FX in the future.

Reply:
I think the Enum is still needed if we want to support other quantization modes in the future.

Reply:
Thanks for clarifying. I now understand that this was done to support other FX-based quantization approaches on the future. :)

Note that we try to keep our examples/reference scripts as simple as possible and that we don't offer strong BC guarantees on this code. So it would be OK in this case for you to just introduce a boolean argument if this greatly simplifies the code. This is just a nit, so definitely no blocking. I'll leave this up to you.

Reply:
by the way, my comments were strictly about the name.  There were some long internal discussions in the past on what to name this product, and in the end "FX graph mode quantization" was the name that was chosen, I was just hoping we can stay consistent to it.  

We also considered "FX quantization" but did not choose that because we wanted to make sure that "graph mode" was in the name.  There is also an outdated "TorchScript graph mode quantization", and an upcoming "DBR quantization" which does not use FX directly but may interact with FX.